inst/rmarkdown/templates/apa6/skeleton/skeleton.Rmd

```diff
@@ -59,6 +59,8 @@ output            : papaja::apa6_pdf
 
 ```{r setup, include = FALSE}
 library("papaja")
+papaja::r_refs("r-references.bib")
```

Comment:
Given that `papaja` is loaded above, we don't really need the `papaja::` prefix here. Also, could you add an empty line between the library call and the call to `r_refs()`?

Reply:
Sorry, one more thing: Could your remove the chunk in lines 95-97, now that the reference file is created up here?